extensions/vertx-http/deployment/src/main/resources/dev-templates/index.html

```diff
@@ -20,23 +20,6 @@
            </div>
        </div>
    </div>
-       <div class="col mb-4">
-           <div class="card h-100 shadow">
-               <div class="card-header bg-primary text-light">
-                   Tests
-                   <a href="https://quarkus.io/guides/getting-started-testing" class="text-light float-right" title="Guide">
-                       <i class="fa fa-book"></i>
-                   </a>
-               </div>
-               <div class="card-body">
-                   <p class="card-text">
-                       <a href="io.quarkus.quarkus-vertx-http/test" class="badge badge-light">
```

Comment:
Didn't we have a `{urlbase}` for this? https://quarkus.io/guides/dev-ui#linking-to-your-full-page-templates

Reply:
This block has been deleted, so not applicable here.

Reply:
urlbase is available in extension that include tiles, but not in this one, that is more internal.... (part of Dev UI)